docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/braze-tag-for-gtm-ss/braze-tag-configuration/index.md

```diff
@@ -14,12 +14,72 @@ Set this to the URL of your Braze REST [endpoint](https://www.braze.com/docs/api
 
 Set this to your Braze [API Key](https://www.braze.com/docs/api/basics/#app-group-rest-api-keys) that will be included in each request. The minimum permission that you need to assign for this API Key is to access the `/users/track` endpoint.
 
-![](images/key_permission.png)
+![key permission](images/key_permission.png)
 
-### Braze external\_id (required)
+### Identity settings
+
+_Note: Since v0.2.0 this section of the configuration has changed. Below you can read about the latest configuration options. If you are interested about pre-0.2.0 versions, there is a section at the end of this one._
+
+#### Braze User Identifier
+
+This section allows you to select which Braze user identifier (external user ID (`external_id`) or [User Alias](https://www.braze.com/docs/api/objects_filters/user_alias_object#user-alias-object-specification) `user_alias`) will be used by the tag. The default value is `external_id`.
+
+![identity settings dropdown](images/identity_settings.png)
+
+##### Braze external\_id
+
+This configuration section is enabled if you have selected the `external_id` as the **Braze User Identifier**.
+
+![braze external id](images/external_id.png)
+
+- **Set external_id from:**
+
+  Use this option to select how you want to set the `external_id`, either from the Event Property you specify or directly from the Value you provide.
+
+- **external\_id**
+
+  Depending on the previous selection, here you can specify the value or the client event property (e.g. client_id) that corresponds to the `external_id` user identifier for Braze API.
+
+##### Braze User Alias Object
+
+This configuration section is enabled if you have selected the `user_alias` as the **Braze User Identifier**.
+
+![braze user alias object](images/user_alias_object.png)
+
+###### Update existing users only
+
+When enabled(default), this option will only update existing users. Uncheck this box to allow creating alias-only users.
+
+###### User Alias Name
+
+- **Set user alias name from:**
+
+  Using this section you can select how you want to set the name of the user alias object: from an Event Property or directly from a Value you provide.
+
+- **Alias Name**
+
+  Depending on the previous selection, here you can specify the value or the client event property that corresponds to the User Alias Name.
+
+###### User Alias Label
+
+- **Set user alias label from:**
+
+  Using this section you can select how you want to set the label of the user alias object: from an Event Property or directly from a Value you provide.
+
+- **Alias Label**
+
+  Depending on the previous selection, here you can specify the value or the client event property that corresponds to the User Alias Label.
+
+:::info
+
+### Pre-v0.2.0
```

Comment:
then I'd suggest collapsing this section.

You can do it with:

```
<details><summary>CLICK ME</summary>
<p>

#### yes, even hidden code blocks!

```python
print("hello world!")
```

</p>
</details>
```

Reply:
(If you go with `<details>`, I supposed you could also update other GTM pages with that. I used `:::info` when converting those pages because I didn’t know about `<details>`.)

Reply:
Doc here: https://docusaurus.io/docs/markdown-features#details

Reply:
Done (also for Amplitude tag). Looks nicer now!